.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'Bug Report: (fill in)'
+labels: ''
+assignees: ''
+
+---
+
+## Expected Behavior
+
+> [Describe what you expected to happen]
```

Comment:
Since this is Markdown, we could replace those parts with HTML comments

It will  still be displayed to the user when filling the issue but won't show when submitted if he forgot to remove them:

```suggestion
<!-- A concise description of what you're experiencing. -->
```

> GitHub as [a great example](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#markdown-issue-template) of one of its template